docs/getting_started.rst

```diff
@@ -0,0 +1,283 @@
+|                             |                 |              |         |                                                                 |
+|                             |                 |              |         | for file information                                            |
++-----------------------------+-----------------+--------------+---------+-----------------------------------------------------------------+
+| :code:`DYAD_PATH_PRODUCER`  | Directory Path  | Yes [#one]_  | N/A     | The producer-managed path of the application                    |
++-----------------------------+                 +              +         +-----------------------------------------------------------------+
+| :code:`DYAD_PATH_CONSUMER`  |                 |              |         | The consumer-managed path of the application                    |
++-----------------------------+-----------------+--------------+---------+-----------------------------------------------------------------+
+| :code:`DYAD_SHARED_STORAGE` | 0 or 1          | No           | 0       | If 1 (i.e., true), only provide per-file synchronization of     |
+|                             |                 |              |         |                                                                 |
+|                             |                 |              |         | the consumer (i.e., no transfer)                                |
++-----------------------------+-----------------+--------------+---------+-----------------------------------------------------------------+
+| :code:`DYAD_KEY_DEPTH`      | Integer         | No           | 3       | The number of levels in Flux's hierarchical KVS to use          |
+|                             |                 |              |         |                                                                 |
+|                             |                 |              |         | within DYAD's namespace                                         |
++-----------------------------+-----------------+--------------+---------+-----------------------------------------------------------------+
+| :code:`DYAD_KEY_BINS`       | Integer         | No           | 1024    | The maximum number of unique values for the keys associated     |
+|                             |                 |              |         |                                                                 |
+|                             |                 |              |         | with any given level of Flux's hierarchical KVS within          |
+|                             |                 |              |         |                                                                 |
+|                             |                 |              |         | DYAD's namespace                                                |
++-----------------------------+-----------------+--------------+---------+-----------------------------------------------------------------+
+
```

Comment:
A comment for the need/purpose of DYAD_KEY. It is for searching performance scalability.